runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -655,6 +671,18 @@ String getClueType()
 		return clueType;
 	}
 
+	@VisibleForTesting
+	String getValuableDropType()
+	{
+		return valuableDropType;
```

Comment:
You can just test that a frame was requested with `verify(drawManager).requestNextFrameListener(Matchers.any(Consumer.class));`. If we really want to start testing filenames then we should have the screenshot-taker be put in a different class which we can mock and then use `verify`.